{"name": "test_filestack_06", "status": "passed", "description": "06. failure case: pop() unexist stuff into stack, an exception raises.", "start": 1758038158004, "stop": 1758038158005, "uuid": "bd8cfe7c-612e-448c-81c1-90d9d16dfbd1", "historyId": "6248c594bb2403538354004f7b76bf06", "testCaseId": "6248c594bb2403538354004f7b76bf06", "fullName": "casatasks.test_task_imbaseline.TestFileStack#test_filestack_06", "labels": [{"name": "parentSuite", "value": "casatasks"}, {"name": "suite", "value": "test_task_imbaseline"}, {"name": "subSuite", "value": "TestFileStack"}, {"name": "host", "value": "casa-rockylinux8-amd64-perf07"}, {"name": "thread", "value": "597558-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "casatasks.test_task_imbaseline"}], "titlePath": ["casatasks", "test_task_imbaseline.py", "TestFileStack"]}